src/components/utils/test/thread_test.cc

```diff
@@ -175,6 +247,175 @@ TEST_F(ThreadTest, StartOneThreadTwice_ExpectTheSameThreadStartedTwice) {
   EXPECT_TRUE(delegate_->check_value());
 }
 
+TEST(ThreadTest2, StartAndJoinForce_Success) {
```

Comment:
@AGritsevich The difference is this test doesn't use a fixture class. Suggest a better name, I think this is fine.